packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -589,33 +599,18 @@ function build(
 export function makeLiveSlots(
   syscall,
   state,
-  buildRootObject,
   forVatID = 'unknown',
   vatPowers = harden({}),
   vatParameters = harden({}),
 ) {
-  const {
-    deliver,
-    notifyFulfillToData,
-    notifyFulfillToPresence,
-    notifyReject,
-  } = build(
-    syscall,
-    state,
-    buildRootObject,
-    forVatID,
-    { ...vatPowers, getInterfaceOf, Remotable, makeMarshal },
-    vatParameters,
-  );
-  return harden({
-    deliver,
-    notifyFulfillToData,
-    notifyFulfillToPresence,
-    notifyReject,
-  });
+  const allVatPowers = { ...vatPowers, getInterfaceOf, Remotable, makeMarshal };
+  const r = build(syscall, state, forVatID, allVatPowers, vatParameters);
+  const { vatGlobals, dispatch, setBuildRootObject } = r; // omit 'm'
+  return harden({ vatGlobals, dispatch, setBuildRootObject });
```

Comment:
Oooh, I like being able to collapse all those multi-line calls into single line calls.

Reply:
yeah, deleting code is aweome :)